tutorials/ui/bbcode_in_richtextlabel.rst

```diff
@@ -90,26 +90,10 @@ Reference
 | **cell**          | ``[cell]{text}[/cell]``                    | Adds cells with the {text} to the table.                        |
 +-------------------+--------------------------------------------+-----------------------------------------------------------------+
 
-Built-in color names
-~~~~~~~~~~~~~~~~~~~~
-
-List of valid color names for the [color=<name>] tag:
-
--  aqua
--  black
--  blue
--  fuchsia
--  gray
--  green
--  lime
--  maroon
--  navy
--  purple
--  red
--  silver
--  teal
--  white
--  yellow
+Named colors
+~~~~~~~~~~~~
+
+You can use the constants of the :ref:`class_Color` class for the ``[color=<name>]`` tag. The case of the name is not important: ``DARK_RED``, ``DarkRed`` and ``darkred`` will give the same result.
```

Comment:
Judging by the description, it's both style-insensitive and case-insensitive :slightly_smiling_face: 

```suggestion
You can use the constants of the :ref:`class_Color` class for the ``[color=<name>]`` tag.
The case and style of the name is not important: ``DARK_RED``, ``DarkRed`` and ``darkred`` will give the same result.
```